src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -199,6 +237,9 @@ internal bool IsValid()
             }
         }
 
+        /// <summary>
+        /// Specifies the range of indices of input columns that should be mapped to an output column.
+        /// </summary>
         public sealed class Range
         {
             public Range() { }
```

Comment:
> public Range() { } [](start = 11, length = 19)

is this used?

Reply:
It is! And I think you are right it looks a bit strange. After discussing with Tom, I am going to open an issue about reshaping the public surface of TextLoader and fix it separately. Let's keep this PR only for internalization and refactoring of Arguments.

---
In reply to: [254397852](https://github.com/dotnet/machinelearning/pull/2417#discussion_r254397852) [](ancestors = 254397852)